### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by HarisPUBG** [ORelio/Minecraft-Console-Client#2823] [Q&A]
how do i make it spam /server survival every 10 seconds // i want it that whenever it disconnects go to hub it types /server survival and after joining too

**Comment by wyf9** [comment]
i think you can use the following config:
```ini
[ChatBot.ScriptScheduler]
Enabled = true

[[ChatBot.ScriptScheduler.TaskList]]
Task_Name = "HeresTaskName"
Trigger_On_First_Login = false
Trigger_On_Login = true
Trigger_On_Times = { Enable = false, Times = [ ] }
Trigger_On_Interval = { Enable = true, MinTime = 10.0, MaxTime = 10.0 }
Action = "send /server survival"
```
**official doc**: https://mccteam.github.io/g/bots/#script-scheduler
